boards/PDM/Src/App/states/App_AirOpenState.c

```diff
@@ -20,6 +20,8 @@ static void AirOpenStateRunOnTick1Hz(struct StateMachine *const state_machine)
 
 static void AirOpenStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
+    App_AllStatesRunOnTick100Hz(state_machine);
```

Comment:
Side-note; if this "AllStates" pattern is super common, it might be worth modifying `SharedStateMachine` to explicitly support it, ie. having `AllStatesRunOnTick100Hz`, `AllStatesRunOnTick1Hz etc. that are called during the respective "tick" functions for the statemachine.

Reply:
Made a ticket for it! 

https://github.com/UBCFormulaElectric/Consolidated-Firmware/issues/649